HEC.FDA.Model/stageDamage/ImpactAreaStageDamage.cs

```diff
@@ -349,18 +352,27 @@ private PairedData IdentifyCentralStageFrequencyAtIndexLocation()
         /// <param name="inventoryAndWaterTupled"></param>
         /// <param name="profileProbabilities"></param>
         /// <returns></returns>
-        private List<StudyAreaConsequencesBinned> ComputeDamageWithUncertaintyAllCoordinates(string damageCategory, (Inventory, List<float[]>) inventoryAndWaterTupled, List<double> profileProbabilities, bool computeIsDeterministic)
+        private List<StudyAreaConsequencesBinned> ComputeDamageWithUncertaintyAllCoordinates(
+            string damageCategory,
+            (Inventory, List<float[]>) inventoryAndWaterTupled,
+            List<double> profileProbabilities,
+            bool computeIsDeterministic,
+            ProgressReporter reporter = null,
+            Stopwatch sw = null)
         {
+            reporter ??= ProgressReporter.None();
 
             //damage for each stage
             List<StudyAreaConsequencesBinned> consequenceDistributionResults = CreateConsequenceDistributionResults(damageCategory);
+            //Debug.WriteLine("Results Count = " + consequenceDistributionResults.Count);
```

Comment:
Go ahead and just nuke this line. I was certainly using it in a debug session, and it never needed committing.